zilencer/management/commands/populate_db.py

```diff
@@ -177,8 +179,22 @@ def handle(self, **options):
                                  Recipient.objects.filter(type=Recipient.STREAM)]
 
         # Extract a list of all users
-        user_profiles = [user_profile.id for user_profile in UserProfile.objects.all()] # type: List[int]
-
+        user_profiles = [user_profile for user_profile in UserProfile.objects.all()] # type: List[Any]
```

Comment:
You can use `user_profiles = list(UserProfile.objects.all())`. The type should be `UserProfile` instead of `Any`.